Feat update architecture

eldadfux opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.

You can learn more about contributing to appwrite here: https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md

Happy contributing!

-->

## What does this PR do?

Updated some of our architecture diagram to apply 0.13 changes and additions.

## Test Plan

N/A

## Related PRs and Issues

N/A

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yep.

TorstenDittmann commented:
In the Overview:
- Realtime is ALL uppercase
- what is that connection from the console to InfluxDB?
- should there also be a connection from StatsD to InfluxDB?

In Cloud Functions:
- I think we can drop the Question in Web IDE
- Maybe rename Code Package to CLI and drop the subtitle?

eldadfux commented:
> In the Overview:
> 
> * Realtime is ALL uppercase - FIXED
> * what is that connection from the console to InfluxDB? - Legacy - FIXED
> * should there also be a connection from StatsD to InfluxDB? - FIXED
> 
> In Cloud Functions: - I deleted this diagram, we'll use the OPR one.
> 
> * I think we can drop the Question in Web IDE
> * Maybe rename Code Package to CLI and drop the subtitle?
